src/DataGrid.tsx

```diff
@@ -974,6 +974,27 @@ function DataGrid<R, SR, K extends Key>(
     );
   }
 
+  function getRowViewportColumns(rowIdx: number) {
```

Comment:
Worth memoizing?

Reply:
Probably not. It is only for the selected row and if the selected idx is not in the viewport